daffodil-japi/src/test/java/org/apache/daffodil/example/TestInfosetOutputter.java

```diff
@@ -51,45 +48,50 @@ public void endDocument() {
     }
 
     @Override
```

Comment:
This shows the actual substantive changes being created here. These no longer have access to DINode classes (at least not without downcasting).